app/controllers/api/v1/packages_controller.rb

```diff
@@ -431,6 +432,11 @@ def assign_detail
         ).build_or_update_record
       end
 
+      def assign_storage_type
+        return unless params["package"]["storage_type"]
```

Comment:
should we maybe have a default storage type for the "normal" package (maybe 'unit') ? 🤔

Reply:
Yep, we can have a default one, but need to decide it's name. Maybe @steveyken can suggest some name for it. 

Also, we can say that if storage_type is nil, it's a package and not anything else. 

I'm open for both. 

Let me know your thoughts on this @patrixr and @steveyken :)

Reply:
As discussed today in the call, I'll create a storage type for "normal" package. By the name of "Package".